tests/testCameraMapper.py

```diff
@@ -413,16 +439,19 @@ def test(self):
         registryA = butler._repos.inputs()[0].repo._mapper.registry
         registryB = butler._repos.outputs()[0].repo._mapper.registry
         self.assertEqual(id(registryA), id(registryB))
+        del registryA
+        del registryB
```

Comment:
I am guessing that these three `del` calls are not needed because the assignment later will overwrite them.